src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -3172,9 +3172,9 @@ private bool HasTopLevelNullabilityConversion(TypeSymbolWithAnnotations source,
                     break;
 
                 case ConversionKind.AnonymousFunction:
-                    if (operandOpt.Kind == BoundKind.Lambda)
+                    if (operandOpt!.Kind == BoundKind.Lambda)
                     {
-                        var lambda = (BoundLambda)operandOpt;
+                        var lambda = (BoundLambda)operandOpt!;
```

Comment:
>operandOpt! [](start = 50, length = 11)

This makes me wonder if we should make `!` sticky. I don't recall if that's still on the open issues list. What do you think?

Reply:
I agree. Logged [https://github.com/dotnet/roslyn/issues/31732](https://github.com/dotnet/roslyn/issues/31732).

---
In reply to: [241228773](https://github.com/dotnet/roslyn/pull/31719#discussion_r241228773) [](ancestors = 241228773)